classes/blocks/class-spreadsheet.php

```diff
@@ -49,9 +48,6 @@ public function __construct() {
 	 * @return array The data to be passed in the View.
 	 */
 	public function prepare_data( $fields ): array {
```

Comment:
If `prepare_data` is used only in the backend render, then you can remove this function, or make it just return an empty array.